test/test_files.rb

```diff
@@ -42,14 +42,16 @@ class TestFiles < Minitest::Test
                                warn("Skipping MathjaxNode tests as MathjaxNode is not available")
                            end
 
-  KATEX_AVAILABLE = begin
+  SSKATEX_AVAILABLE = begin
```

Comment:
The following lines should be indented two additional spaces since `SSKATEX_AVAILABLE` is two characters longer than `KATEX_AVAILABLE`.

Reply:
Done